collect_app/src/main/java/org/odk/collect/android/fragments/ShowQRCodeFragment.java

```diff
@@ -149,7 +150,14 @@ public void onClick(View v) {
             case R.id.btnSelect:
                 Intent photoPickerIntent = new Intent(Intent.ACTION_PICK);
                 photoPickerIntent.setType("image/*");
-                startActivityForResult(photoPickerIntent, SELECT_PHOTO);
+                if (photoPickerIntent.resolveActivity(getActivity().getPackageManager()) != null) {
+                    startActivityForResult(photoPickerIntent, SELECT_PHOTO);
+                } else {
+                    Toast.makeText(getActivity(),
+                            getString(R.string.activity_not_found, "select image"),
+                            Toast.LENGTH_SHORT).show();
+                    Timber.w(getString(R.string.activity_not_found, "select image"));
```

Comment:
Is `getString` required in Timber? I guess this will also work
`Timber.w(R.string.activity_not_found, "select image");`